example/src/main/java/com/stripe/example/module/RetrofitFactory.java

```diff
@@ -0,0 +1,47 @@
+package com.stripe.example.module;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Factory to generate our Retrofit instance.
+ */
+public class RetrofitFactory {
+
+    // Put your Base URL here. Unless you customized it, the URL will be something like
+    // https://hidden-beach-12345.herokuapp.com/
+    private static final String BASE_URL = "put your url here";
```

Comment:
will we have a documentation somewhere to explain to users how to set this up with their own backend?